classes/image_types_fsl.bbclass

```diff
@@ -42,3 +42,6 @@ IMAGE_CMD_wic_append_mxs() {
 	bbnote "Setting partition type to 0x53 as required for mxs' SoC family."
 	echo -n S | dd of=$out${IMAGE_NAME_SUFFIX}.wic bs=1 count=1 seek=450 conv=notrunc
 }
+
+# Include required software for optee
+IMAGE_INSTALL += "${@bb.utils.contains('MACHINE_FEATURES', 'optee', 'packagegroup-fsl-optee-imx', '', d)}"
```

Comment:
Please move this to `imx-base.inc` and add it to `MACHINE_ESSENTIAL_RRECOMMENDS` or similar.